chroma_core/models/stratagem.py

```diff
@@ -104,10 +131,15 @@ def run(self, kwargs):
                 "After=iml-manager.target\n"
                 "\n[Service]\n"
                 "Type=oneshot\n"
-                "ExecStart={}\n".format(config.filesystem_id, iml_cmd)
+                "ExecStart={}\n".format(config.filesystem.id, iml_cmd)
             )
         self.try_shell(["systemctl", "daemon-reload"])
         self.try_shell(["systemctl", "enable", "--now", "{}.timer".format(name)])
+        
+        # This step happens so quickly that sometimes the modal doesn't have an opportunity to appear. 
+        # Adding a short sleep will ensure that the command modal can be launched. This is temporary 
+        # for the demo and until we can find a better solution.
+        time.sleep(1)
```

Comment:
This is a front-end problem. Why is there a possibility that the modal can be missed?